frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import { useValues } from 'kea'
+import { SceneExport } from 'scenes/sceneTypes'
+import { urls } from 'scenes/urls'
+import { PageHeader } from 'lib/components/PageHeader'
+import { DataManagementPageTabs, DataManagementTab } from 'scenes/data-management/DataManagementPageTabs'
+import { IngestionWarning, ingestionWarningsLogic, IngestionWarningSummary } from './ingestionWarningsLogic'
+import { LemonTable } from 'lib/components/LemonTable'
+import { TZLabel } from 'lib/components/TimezoneAware'
+import { Link } from 'lib/components/Link'
+
+export const scene: SceneExport = {
+    component: IngestionWarningsView,
+    logic: ingestionWarningsLogic,
+}
+
+const WARNING_TYPE_TO_DESCRIPTION = {
+    cannot_merge_already_identified: 'Refused to merge an already identified user via $identify or $create_alias call',
+}
+
+const WARNING_TYPE_RENDERER = {
+    cannot_merge_already_identified: function Render(warning: IngestionWarning): JSX.Element {
+        const details = warning.details as {
+            sourcePerson: string
+            sourcePersonDistinctId: string
+            targetPerson: string
+            targetPersonDistinctId: string
+        }
+        return (
+            <>
+                Refused to merge already identified person{' '}
+                <Link to={urls.person(details.sourcePersonDistinctId)}>{details.sourcePersonDistinctId}</Link> into{' '}
+                <Link to={urls.person(details.targetPersonDistinctId)}>{details.targetPersonDistinctId}</Link> via an
+                $identify or $create_alias call
+            </>
+        )
+    },
+}
+
+export function IngestionWarningsView(): JSX.Element {
+    const { data, dataLoading } = useValues(ingestionWarningsLogic)
+
+    return (
+        <div data-attr="manage-events-table">
+            <PageHeader
+                title="Data Management"
+                caption="Use data management to organize events that come into PostHog. Reduce noise, clarify usage, and help collaborators get the most value from your data."
+                tabbedPage
+            />
+            <DataManagementPageTabs tab={DataManagementTab.IngestionWarnings} />
+
+            <div className="mb-4">Data ingestion related warnings from past 30 days.</div>
+
+            <LemonTable
```

Comment:
In addition to a table how hard would it be to add a graph (not customisable if easier) of warnings over time grouped by warning type at the top?

Reply:
You mean like sentry? Not sure, would need to dig into chartjs a bit, probably 20 minutes to 2 hours.

Reply:
yup or our charts sth like https://app.posthog.com/insights/71mRDVNa - which is async migration errors grouped by name